### ISSUE TITLE
make distcheck: file name is too long

### ISSUE THREAD

**Issue by dongahn** [flux-framework/flux-sched#601]
```
make[2]: Leaving directory '/usr/src/t'
test -n "" \
|| find "flux-sched-0.8.0-42-gb5a437bf" -type d ! -perm -755 \
	-exec chmod u+rwx,go+rx {} \; -o \
  ! -type d ! -perm -444 -links 1 -exec chmod a+r {} \; -o \
  ! -type d ! -perm -400 -exec chmod a+r {} \; -o \
  ! -type d ! -perm -444 -exec /bin/bash /usr/src/config/install-sh -c -m a+r {} {} \; \
|| chmod -R a+r "flux-sched-0.8.0-42-gb5a437bf"
tardir=flux-sched-0.8.0-42-gb5a437bf && ${TAR-tar} chof - "$tardir" | eval GZIP= gzip --best -c >flux-sched-0.8.0-42-gb5a437bf.tar.gz
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_missing_count.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_count_missing_min.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_count_missing_operator.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_count_missing_max.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resources_not_sequence.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_exclusive_invalid.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_exclusive_not_scalar.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/task_count_not_mapping.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_count_bad_type.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_count_scalar_bad_value.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/attributes_not_mapping.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_id_not_scalar.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_unit_not_scalar.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/task_command_not_array.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_count_missing_operand.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_label_not_scalar.yaml: file name is too long (max 99); not dumped
tar: flux-sched-0.8.0-42-gb5a437bf/t/data/resource/jobspecs/validation/invalid/resource_slot_not_labelled.yaml: file name is too long (max 99); not dumped
tar: Exiting with failure status due to previous errors
```

make distcheck generates the above message.

**Comment by dongahn** [comment]
https://noiselabs.io/tar-file-name-is-too-long-max-99/

```
AM_INIT_AUTOMAKE([1.9 tar-pax])
```

might be a fix.

**Comment by dongahn** [comment]
This has been fixed by PR #677
